EasyPost.Net/Order.cs

```diff
@@ -85,7 +85,7 @@ public void Create()
         /// <summary>
         ///     Populate the rates property for this Order.
         /// </summary>
-        public void GetRates()
+        public void RegenerateRates(Dictionary<string, object> parameters = null)
```

Comment:
You can't regenerate rates for an order, only a shipment. You'll want to put this back to what it was in both locations.